### ISSUE TITLE
Logo

### ISSUE THREAD

**Issue by mxstbr** [carteb/carte-blanche#342]
We need a proper logo. Nik's [quickly made for launch one](https://avatars1.githubusercontent.com/u/15898536?v=3&s=200) is alright, but I want some stickers with a nice logo!

/cc @elioqoshi @okonet who've helped before with logos!

**Comment by FezVrasta** [comment]
Do we want to refer to React in the logo?

**Comment by okonet** [comment]
I don't think so, to be honest. Especially keeping in mind that carte-blanche has plan to support other frameworks/libs.

**Comment by elioqoshi** [comment]
I'd be happy to take this on :)

**Comment by mxstbr** [comment]
That'd be awesome!

**Comment by elioqoshi** [comment]
What do you think about this? @mxstbr 
![carte blanche](https://cloud.githubusercontent.com/assets/5436686/16177815/b4f3ab1a-3637-11e6-96d1-0be878a322b6.jpg)

**Comment by eddiewang** [comment]
i'll also make one tonight.

**Comment by nikgraf** [comment]
yay, looking forward to a couple of variations 🙌 

@elioqoshi nice one, what I recommend in general is to not use uppercase characters all the way (which is funny, because yet I couldn't even convince my own team 😜)

**Comment by elioqoshi** [comment]
@nikgraf why specifically? There is no thumb rule as in "avoid doing x" in that case so is there any reason for that here apart personal preference?

**Comment by okonet** [comment]
@elioqoshi it is about preference (I guess) but I also think that in most cases it makes it harder to read. I realize logos aren't about reading, but I'd also would like to see a version without all caps.

**Comment by mxstbr** [comment]
I'm not sure I understand what the icon means, how is it related to Carte Blanche? Looks like an "N" to me, kind of invokes feeling of a construction company…

**Comment by elioqoshi** [comment]
@okonet makes sense

@mxstbr it's supposed to be 2 cards stacked against each other (like you would do with a card house). Seems it's not too obvious...

**Comment by nikgraf** [comment]
"Studies have been conducted on the readability and legibility of all caps text. Scientific testing from the 20th-century onwards has generally indicated that all caps text is less legible and readable than lower-case text.[2][3] In addition, switching to all caps may make text appear hectoring and obnoxious for cultural reasons, since all-capitals is often used in transcribed speech to indicate that the speaker is shouting." https://en.wikipedia.org/wiki/All_caps

@elioqoshi To me readability in a logo is not too important to me as it is not a long text to read, but the second part doesn't "hectoring and obnoxious" is not reflecting what I like this tool be perceived. How do you feel about it?

**Comment by ac360** [comment]
I should probably get back to work.  But spent a few minutes trying to be helpful.

![temp](https://cloud.githubusercontent.com/assets/2752551/16250078/6b605748-37cf-11e6-97eb-e45293b88d2c.png)

**Comment by eddiewang** [comment]
Here's a possible version. Have alot of work so maybe will have more designs later.
![carte blanche](https://cloud.githubusercontent.com/assets/2057997/16250732/c845aca6-37ed-11e6-9a31-36035afa18bb.png)

**Comment by paulxuca** [comment]
![carte](https://cloud.githubusercontent.com/assets/7885473/16279999/81e32fb0-388b-11e6-9e97-a80f6ad4b57a.jpg)

**Comment by okonet** [comment]
@paulxuca I like the idea with cards but I think the opacity won't work well on most screens. It's hard to see the last card even on the iPhone. It also look to similar to the Medium's logo IMO.

**Comment by paulxuca** [comment]
![carte](https://cloud.githubusercontent.com/assets/7885473/16282899/a08f9f68-3898-11e6-838c-696f170edec2.jpg)
different approach (playing on the idea of a blanche card and testing components).

**Comment by mxstbr** [comment]
I like where this is going! I like the colors in @ac360, though I'm not sure a whisky glass is what we want as an icon :grin: @eddiewang and @paulxuca's direction is great!

The last @paulxuca one is a bit too explicit in my opinion, there's too much fidelity in the icon.

**Comment by okonet** [comment]
Here is my (very rough) attempt.

It is based on a geometric form of 2 cards (what else!) and first capital letters of Carte Blanche. First variant resembles similarity with Node.js (JavaScript).

![2016-06-23 at 12 11](https://cloud.githubusercontent.com/assets/11071/16299661/be30aad2-393b-11e6-9fdb-a660134b142c.png)

Both are quick sketches and should be cleaned up in case we go with them. I'm also not completely sold on the type yet but I think for the GitHub project this isn't important since we can't change it on the GH page itself and only the sign will appear (I guess).

**Comment by okonet** [comment]
A few more ideas:

![2016-06-23 at 16 10](https://cloud.githubusercontent.com/assets/11071/16306180/23bda348-395d-11e6-9f1e-b38c5379677f.png)

**Comment by mxstbr** [comment]
I like the right one in your last comment @okonet, but the colors seem a bit… muddy? The `b` is also a bit hard to see.

**Comment by elioqoshi** [comment]
What do you think about this?

![1-01](https://cloud.githubusercontent.com/assets/5436686/16539293/e8293818-403e-11e6-9a3e-6f6b76f3c250.jpg)
